### PR TITLE
[SHRINKRES-320] Resolve control flow issues

### PR DESCRIPTION
**Pull request by petrberan**
https://issues.redhat.com/browse/SHRINKRES-320

### CODE REVIEW

File: maven/api-maven/src/main/java/org/jboss/shrinkwrap/resolver/api/maven/coordinate/MavenCoordinateImpl.java [hunk 1]
```diff
@@ -157,10 +157,7 @@ public boolean equals(final Object obj) {
         } else if (!classifier.equals(other.classifier)) {
             return false;
         }
-        if (!packaging.equals(other.packaging)) {
-            return false;
-        }
-        return true;
+        return (packaging.equals(other.packaging));
```
Comment (xstefank) [inline maven/api-maven/src/main/java/org/jboss/shrinkwrap/resolver/api/maven/coordinate/MavenCoordinateImpl.java:160]: outer () not needed

File: maven/api-maven/src/main/java/org/jboss/shrinkwrap/resolver/api/maven/coordinate/MavenGABaseImpl.java [hunk 2]
```diff
@@ -117,13 +117,8 @@ public boolean equals(final Object obj) {
...
         if (groupId == null) {
-            if (other.groupId != null) {
-                return false;
-            }
-        } else if (!groupId.equals(other.groupId)) {
-            return false;
-        }
-        return true;
+            return other.groupId == null;
+        } else return (groupId.equals(other.groupId));
```
Comment (xstefank) [inline maven/api-maven/src/main/java/org/jboss/shrinkwrap/resolver/api/maven/coordinate/MavenGABaseImpl.java:121]: same

**Review by xstefank — CHANGES_REQUESTED** [review]
Fix the parentheses, please

**Comment by petrberan** [comment]
Fixed everywhere

**Review by xstefank — APPROVED** [review]
LGTM

**Pull request merged**